### PR TITLE
FINERACT-835: Complete work on automatically generating Swagger docs

### PR DESCRIPTION
**Pull request by ptuomola**
## Description
This PR fixes the Swagger UI versioning issue and includes the remaining metadata from old response.json to the new autogenerated fineract.json. It also removes the old response.json and completes the work on the following JIRA issues:
- FINERACT-835
- FINERACT-837
- FINERACT-1045
- FINERACT-1083

## Checklist
Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] Commit message starts with the issue number from https://issues.apache.org/jira/projects/FINERACT/. Ex: FINERACT-646 Pockets API.

- [x] Coding conventions at https://cwiki.apache.org/confluence/display/FINERACT/Coding+Conventions have been followed.

- [x] API documentation at fineract-provider/src/main/resources/static/api-docs/apiLive.htm has been updated with details of any API changes.

- [x] Integration tests have been created/updated for verifying the changes made.

- [x] All Integrations tests are passing with the new commits.

- [x] Submission is not a "code dump".  (Large changes can be made "in repository" via a branch.  Ask on the list.)

Our guidelines for code reviews is at https://cwiki.apache.org/confluence/display/FINERACT/Code+Review+Guide

### CODE REVIEW

**Comment by thesmallstar** [comment]
@ptuomola this is failing due to minor spotless violations, spotlessApply should fix this :D

**Review by thesmallstar — APPROVED** [review]
GREAT! this works as expected!!

**Comment by thesmallstar** [comment]
@ptuomola is there any reason you used JSON over yaml for this config? 
All the docs of swagger-ui use yaml, would you mind if I convert it to yaml in a PR following this (which will fix the needed config for try it out button).

**Comment by ptuomola** [comment]
@thesmallstar No reason - just that the first examples I found for the config file format were in JSON, so I followed them. Feel free to change to YAML once this PR has been merged...

**Comment by thesmallstar** [comment]
Sounds good :D 
@awasum @xurror @Sanyam96  can you review/merge this, please?

File: fineract-provider/config/swagger/fineract-input.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+    "termsOfService": "https://demo.fineract.dev/fineract-provider/api-docs/apiLive.htm",
```
Comment (vorburger) [inline fineract-provider/config/swagger/fineract-input.json:7]: Is it possible to just leave this out entirely? Or use /fineract-provider/... without fineract.dev hostname?

**Comment by vorburger** [comment]
> GREAT! this works as expected!! 

@thesmallstar can I double check, have you actually pulled this PR locally manually tested it yourself? Once you confirm, I'll just merge this, without further testing it myself first... PS: I love seeing the collaboration between you and @ptuomola here!

**Comment by thesmallstar** [comment]
@vorburger  yes I have pulled this and tested it. 
Also, I could make PR #1169 work locally only on this change :)

File: fineract-provider/config/swagger/fineract-input.json (hunk 1 above)
Comment (ptuomola) [reply fineract-provider/config/swagger/fineract-input.json:7]: For the terms of service-attribute specifically? I think it's optional and also a relative path should work. @thesmallstar was going to convert this file to YAML in a separate PR, so maybe good idea to leave it out from that version - I don't think the API docs really qualify as a "terms of service" anyway?

**Comment by Sanyam96** [comment]
Thanks @ptuomola @thesmallstar @vorburger for the implementation and merging this.

**Pull request merged**